tests/framework/pkg/nodes/nodes.go

```diff
@@ -24,13 +24,15 @@ type SSHPath struct {
 	SSHPath string `json:"sshPath" yaml:"sshPath"`
 }
 
-// Node is a configuration of node that is from an oudise cloud provider
+// Node is a configuration of node that is from an outside cloud provider
 type Node struct {
-	NodeID          string `json:"nodeID" yaml:"nodeID"`
-	PublicIPAddress string `json:"publicIPAddress" yaml:"publicIPAddress"`
-	SSHUser         string `json:"sshUser" yaml:"sshUser"`
-	SSHKeyName      string `json:"sshKeyName" yaml:"sshKeyName"`
-	SSHKey          []byte
+	NodeID           string `json:"nodeID" yaml:"nodeID"`
+	PublicIPAddress  string `json:"publicIPAddress" yaml:"publicIPAddress"`
+	PrivateIPAddress string `json:"privateIPAddress" yaml:"privateIPAddress"`
+	SSHUser          string `json:"sshUser" yaml:"sshUser"`
+	SSHKeyName       string `json:"sshKeyName" yaml:"sshKeyName"`
+	SSHKey           []byte
+	IsWindows        bool `json:"isWindows" yaml:"isWindows"`
```

Comment:
Question: are we not going to have the default be false like you're doing in the `config.go` file?

Reply:
I did not find it necessary to add it again here, since it is already set in the config. Do you think I need it in both places?

Reply:
I don't think it's necessary, more of a clarification question that I had.

Reply:
If it's not in the config file does it default to false?

Reply:
if there is ec2Config defined, `isWindows` will default to false if it is not defined. if ec2Config is not defined, isWindows will not be defined.